appengine/java/appengine_runner.sh.template

```diff
@@ -78,6 +78,7 @@ ARGS=(
   "${JVM_FLAGS_CMDLINE[@]}"
   "-Dappengine.sdk.root=${APP_ENGINE_ROOT}"
   ${main_class}
+  "--disable_update_check"
```

Comment:
why?

Reply:
It's the java counterpart to this change to the python file.
https://github.com/bazelbuild/rules_appengine/pull/56/files#diff-b042c7a50d680fc8c3189481231c70e5R134